cli/test/mockcli.go

```diff
@@ -66,28 +66,48 @@ func (c *mockCLIClient) run(cmd []string, params []string, args []string) string
 	return str
 }
 
-func (c *mockCLIClient) runCmd(input []string) string {
-	cmd := c.cmdByNameSub(input[0], input[1])
-	out, err := c.runCmdRaw(cmd, input[2:])
+func (c *MockCLIClient) RunCmd(input []string) string {
```

Comment:
Could be
```suggestion
func (c *MockCLIClient) RunCmd(input ...string) string {
```
So we could avoid constructing those `cmd = []string{..}` arrays everywhere